index.html

```diff
@@ -15,19 +15,23 @@ <h2>Overview</h2>
          <table>
     <tr>
         <td><img src="https://mk0itemsssrcorgqb419.kinstacdn.com/wp-content/uploads/2019/03/iStock-165790610-1000x569.jpg" width="350" height="300"></td>
-        <td><h2>What is Thinkment?</h2> <p>Thinkment is a philosphical and ideology based website that focuses on describing the pros/cons, political influence, systematic practice, and connections between Socialism and Capitalism being controversial topics. Philosophy will further look at the influence of the age of enlightment, the Roman influence of early Afro-European politics and nationalites.</p></td>
+        <td><h2>What is Thinkment?</h2> <p>Thinkment is a philosphical and ideological based website that focuses on describing the pros/cons, political influence, systematic practice, and connections between Socialism and Capitalism being controversial topics. Philosophy will further look at the influence of the age of enlightment, the Roman influence of early Afro-European politics and nationalites.</p></td>
```

Comment:
keep this as ideology. `ideology based` makes sense in english, `ideological based` is not correct english. Either that or get rid of `based` after `ideological`.

` pros/cons` -> `pros and cons` in prose you don't want to use shorthands like \

Also, what is Philosophy in this sentence? `Philosophy will further look` . Is Philosophy an actor of some kind? A person? What do you mean to actually say here?